_posts/2017-10-20-the-data-and-science-behind-grabshare-part-i.md

```diff
@@ -0,0 +1,117 @@
+
+1. A passenger makes a booking; 
+
+2. A GrabCar driver is assigned to the booking; 
+
+3. The assigned driver picks up the passenger and ferries him/her to the destination and the ride is completed. 
+
+It is common for people to arrange for manual ride-sharing with our friends traveling in the same direction to save on travel cost as well as to socialize and connect during the trip. By making use of real-time integrated ride information in the Grab system, we aimed to automatically match strangers traveling in similar directions and assign the same vehicle to both their journeys, allowing them to effectively car-pool. Before promoting the concept of GrabShare however, we had to verify its potential from the existing GrabCar bookings. For example, during morning peak hours we mappped every single booking into a four-dimensional vector with the latitudes and longitudes of pickup and drop-off locations. In addition, the latitudes and longitudes were transformed into a Universal Transverse Mercator (UTM) format to map the earth’s surface to an 2-dimensional Cartesian Coordinate System for distance calculation. After applying a DBSCAN cluster method [[10]](#10) with parameter "eps=300", which means that only bookings with distance of less than 300 meters can be considered as neighbourhoods, we observed eight clear clusters of booking with close pickup and drop-off locations in Figure 1. 
+
+![booking clustering](/img/the-data-and-science-behind-grabshare-part-i/booking-clustering.png)
+*Figure 1. Morning booking clusters with similar itineraries*
+
+The booking requests within each cluster can be allocated and fulfilled with less vehicles, through pooling. Even though not all of them may be willing to share vehicles with others, at least those with unallocated bookings (around 8%) may benefit. After repeating this analysis for different time periods, we observed that a certain percentage of the bookings could be covered with good performing clusters as seen in Table I. We observed that the coverage rate for different time periods fluctuates from 35% to 45% for most part of the day (coverage during mid-night and early morning hours is much smaller as the amount of bookings is much smaller). Because bookings in the same cluster are "near perfect matches" with very close pickup and drop-off location, the potential for GrabShare was found to be quite promising because we could expect even more opportunities for matching in the middle of a trip.
+
+<table class="center">
```

Comment:
I think there should be a way to use Markdown syntax for table and yet get it center-aligned. Have you tried that?

Reply:
tried, it would still render the table, but anw i need to add css to center it, so using html i can add class which is better than using element name.